Dockerfile

```diff
@@ -17,39 +17,39 @@ RUN echo "alias lla='ls -lAhG --color=auto'" >> ~/.bashrc
 WORKDIR /root
 
 
-# google-chrome-stable
-
-
 # ============== DART ==============
 # See https://github.com/dart-lang/dart-docker
-# See https://github.com/dart-lang/setup-dart/blob/main/setup.sh
 FROM base as dart
-ARG DART_VERSION=latest
 ARG DART_CHANNEL=stable
-ENV DART_VERSION=$DART_VERSION
 ENV DART_CHANNEL=$DART_CHANNEL
 ENV DART_SDK=/usr/lib/dart
 ENV PATH=$DART_SDK/bin:$PATH
 RUN set -eu; \
     case "$(dpkg --print-architecture)_${DART_CHANNEL}" in \
       amd64_stable) \
         DART_SHA256="05fba372d64932dffce90bbd45116b76806bf9adc6203967b56faf5c64b2b66c"; \
-        SDK_ARCH="x64";; \
+        SDK_ARCH="x64"; \
+        DART_VERSION="3.0.5";; \
```

Comment:
I'd suggest putting these as versions above to separate configuration from code.